packages/flutter_tools/lib/src/cache.dart

```diff
@@ -890,6 +912,39 @@ class MacOSFuchsiaSDKArtifacts extends _FuchsiaSDKArtifacts {
   }
 }
 
+/// Cached iOS/USB binary artifacts.
+class IosUsbArtifacts extends CachedArtifact {
+  IosUsbArtifacts(String name, Cache cache) : super(
+    name,
+    cache,
+    const <DevelopmentArtifact>{ DevelopmentArtifact.iOS },
```

Comment:
Note: this should be DevelopmentArtifact.universal, and documented that it is required since we cannot detect that an iOS device is connected without these artifacts